packages/frontend-api/src/Model/Resolver/Pagination/NodeResolver.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Pagination;
+
+use Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface;
+use Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface;
+use Overblog\GraphQLBundle\Relay\Connection\Output\Edge;
+use Shopsys\ShopBundle\Model\Product\Product;
+
+class NodeResolver implements ResolverInterface, AliasedInterface
+{
+    /**
+     * @param \Overblog\GraphQLBundle\Relay\Connection\Output\Edge $edge
+     * @return mixed|Product
```

Comment:
You should use FQN. And remove Product from ShopBunde in use blocks :smile:

Reply:
You are right. Fixed

Reply:
Hello, there should not be a dependency on any entity (in this case on product entity). If you will add some blog pagination or orders pagination then you will always edit this line?